frontend/src/metabase/visualizations/components/LineAreaBarChart.css

```diff
@@ -7,8 +7,9 @@
 }
 
 .LineAreaBarChart .dc-chart .grid-line.horizontal {
-  stroke: color-mod(var(--color-text-medium) alpha(-80%));
-  stroke-dasharray: 5, 5;
+  stroke: var(--color-text-medium) !important;
```

Comment:
I think something about our css load order was causing these to just use the default dc.js styling.